### PR TITLE
HPCC-14977 Show logical file protection

### PR DESCRIPTION
**Pull request by miguelvazq**
Provide a visual way of determining if a logical file is locked or not.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

File: esp/src/eclwatch/LFDetailsWidget.js [hunk 1]
```diff
@@ -324,6 +324,9 @@ define([
                 this.updateInput("DespraySourceName", oldValue, newValue);
                 this.updateInput("CopySourceName", oldValue, newValue);
                 this.updateInput("CopyTargetName", oldValue, newValue);
+            } else if (name === "IsProtected") {
+                dom.byId(this.id + "ProtectedImage").src = this.logicalFile.getProtectedImage();
+                console.log(this.logicalFile.getProtectedImage())
```
Comment (GordonSmith) [inline esp/src/eclwatch/LFDetailsWidget.js]: remove console.log

**Comment by GordonSmith** [comment]
One console.log to be removed, otherwise looks fine.

**Comment by miguelvazq** [comment]
@richardkchapman Should be good now.

**Comment by miguelvazq** [comment]
@g-pan This needs screenshots.

**Pull request merged**